myhpom/models/cloudfactory.py

```diff
@@ -2,145 +2,116 @@
 import json
 import datetime
 from django.db import models
-
-
-class CloudFactoryRun(models.Model):
-    """Store information about current and past CloudFactory runs."""
-
-    line_id = models.CharField(
-        max_length=64, help_text="the id of the production line at CloudFactory."
+from django.core.urlresolvers import reverse
+from django.conf import settings
+from myhpom.models import DocumentUrl
+
+# from myhpom.models.user import User
+
+STATUS_NEW = 'NEW'
+STATUS_DELETED = 'DELETED'
+STATUS_PROCESSING = 'Processing'
+STATUS_PROCESSED = 'Processed'
+STATUS_VALUES = [STATUS_NEW, STATUS_DELETED, STATUS_PROCESSING, STATUS_PROCESSED]
+STATUS_MAX_LENGTH = 16
+STATUS_CHOICES = [(i, i) for i in STATUS_VALUES]
```

Comment:
I like to put these into the models, as its clearer what the constants are associated with (referencing module `run.status = cloudfactory.STATUS_NEW` vs referencing model `run.status = CloudFactoryDocumentRun.STATUS_NEW`...particularly when there a module starts having multiple models in a module).

Reply:
Good idea.